unblob/cli.py

```diff
@@ -74,7 +74,11 @@ def get_help_text():
 )
 def cli(files: Tuple[Path], extract_root: Path, depth: int, verbose: bool):
     configure_logger(verbose, extract_root)
-    logger.info("Start processing files", count=noformat(len(files)))
+    file_count = len(files)
+    if file_count <= 0:
```

Comment:
I'd just set `required=True` for the files argument

Reply:
Yeah, good idea!